website/docs/cdktf/concepts/resources.mdx

```diff
@@ -917,3 +917,13 @@ new SecurityGroup(this, "security2", new SecurityGroupConfig
 ```
 
 </CodeTabs>
+
+## Special Cases
+
+### Large Resource Configurations
+
+A few individual Terraform Resources have very deeply nested schemas with a lot of attributes. This blows up the config classes and slows down the code generation for languages besides Typescript. To work around this we sometimes limit the depth of the config classes and use `any` on deeper level, some attributes we directly expose as `any` on the top level config class.
```

Comment:
Might want an example of how to use them. Particularly in non-Typescript as working with `any` is quite tricky.

Reply:
Yeah good point, I'll add some in a separate PR 👍